beamer/events.py

```diff
@@ -177,6 +177,9 @@ def _decode_event(
         kwargs["chain_id"] = chain_id
         kwargs["block_number"] = log_entry["blockNumber"]
         kwargs["tx_hash"] = log_entry["transactionHash"]
+        fill_id = kwargs.get("fill_id")
+        if fill_id is not None:
+            kwargs["fill_id"] = FillId(HexBytes(fill_id))
```

Comment:
FillId is not a NewType of HexBytes. It's a NewType of bytes
see https://github.com/beamer-bridge/beamer/blob/c5aa401c4984993084a561492ed36bd14ad3f8d9/beamer/typing.py#L13

Reply:
You're totally right, this had a couple of iterations and the PR description as well as the content is just wrong.
Pushed another version.